avatar/avatar.html

```diff
@@ -0,0 +1,277 @@
+                </div>
+                <div class="content-subtitle">
+                    <b>Avatar displays user image or initials as profile pictures.</b> 
+                <br>
+                    You can create variety of avatars, including Circle, Square and Text Avatars. 
+                <br>
+                    These can be utilized in five sizes: extra small, small, medium, large, and extra large.
+                </div>
+                
+
+                <div class="divider"></div>
+
+                <!-- CIRLCE AVATAR -->
+                <div class="content-head">
+                    Circle Avatars
+                </div>
+
+                <div class="content-subtitle">
+                    To generate a circle avatar use the base class <b>avatar</b>.
+                    <br>
+                    By default a medium sized, circle avatar is generated. For a varied size, additionaly add the size class (xlarge, large, medium, small, xsmall) as per the requirement.
+                </div>
+
+                <div class="example-display">
+                    <img class="avatar xlarge" src="/images/avatar1.jpg" alt="xlarge size avatar">
+                    <img class="avatar large" src="/images/avatar1.jpg" alt="large size avatar">
+                    <img class="avatar medium" src="/images/avatar1.jpg" alt="medium size avatar">
+                    <img class="avatar small" src="/images/avatar1.jpg" alt="small size avatar">
+                    <img class="avatar xsmall" src="/images/avatar1.jpg" alt="xsmall size avatar">
+                </div>
+
+                <div class="content-subtitle">
+                    Use the below code for circle avatars in different sizes. 
+                </div>
+
+                <div class="code-container">
+                    <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=one-light&wt=none&l=css&width=800&ds=true&dsyoff=100px&dsblur=100px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%252F*Image%2520circular%2520avatars%2520in%2520different%2520sizes*%252F%250A%253Cimg%2520class%253D%2522avatar%2520xlarge%2522%2520src%253D%2522%252Fimages%252Favatar1.jpg%2522%2520alt%253D%2522xlarge%2520size%2520avatar%2522%253E%250A%253Cimg%2520class%253D%2522avatar%2520large%2522%2520src%253D%2522%252Fimages%252Favatar1.jpg%2522%2520alt%253D%2522large%2520size%2520avatar%2522%253E%250A%253Cimg%2520class%253D%2522avatar%2520medium%2522%2520src%253D%2522%252Fimages%252Favatar1.jpg%2522%2520alt%253D%2522medium%2520size%2520avatar%2522%253E%250A%253Cimg%2520class%253D%2522avatar%2520small%2522%2520src%253D%2522%252Fimages%252Favatar1.jpg%2522%2520alt%253D%2522small%2520size%2520avatar%2522%253E%250A%253Cimg%2520class%253D%2522avatar%2520xsmall%2522%2520src%253D%2522%252Fimages%252Favatar1.jpg%2522%2520alt%253D%2522xsmall%2520size%2520avatar%2522%253E"
+                    style="height: 186px;"
```

Comment:
Please avoid inline styles. You can create a separate class for carbon iframes, copy from here and paste it there.